src/Spinner.php

```diff
@@ -55,6 +55,8 @@ public function spin(Closure $callback): mixed
             exit();
         });
 
+        $pid = 0;
```

Comment:
Inititalize the `$pid` before the `try` in case `$this->render()` fails.

Reply:
`0` has a special meaning when passed to `posix_kill`:

![image](https://github.com/laravel/prompts/assets/4977161/90f36226-2e9c-4b49-8aa9-903c92100a75)

I'm not entirely sure of the implications, so I think it would be preferable to remove this line and handle it not being set.